modules/reactor/src/meshgenerators/PatternedCartesianMeshGenerator.C

```diff
@@ -428,10 +428,17 @@ PatternedCartesianMeshGenerator::generate()
     if (!MooseUtils::absoluteFuzzyEqual(
             *std::max_element(pattern_pitch_array.begin(), pattern_pitch_array.end()),
             *std::min_element(pattern_pitch_array.begin(), pattern_pitch_array.end())))
-      mooseError("In PatternedCartesianMeshGenerator ",
-                 _name,
-                 ": pattern_pitch metadata values of all input mesh generators must be identical "
-                 "when pattern_boundary is 'none' and generate_core_metadata is true.");
+      mooseError(
+          "In PatternedCartesianMeshGenerator ",
+          _name,
```

Comment:
you dont need those lines, for future reference. They are already included in mooseError